src/nvim/option.c

```diff
@@ -900,11 +900,18 @@ set_option_default(
       if (options[opt_idx].indir == PV_SCROLL) {
         win_comp_scroll(curwin);
       } else {
-        *(long *)varp = (long)(intptr_t)options[opt_idx].def_val[dvi];
+        long def_val = (long)options[opt_idx].def_val[dvi];
+        if ((long *)varp == &curwin->w_p_so
+                || (long *)varp == &curwin->w_p_siso)
+            // 'scrolloff' and 'sidescrolloff' local values have a
```

Comment:
indentation (should be same as next line)

Reply:
Referring to line 905?

Reply:
Referring to the line at which I commented. Though the issue is actually that TAB:s are used on the subsequent lines, replace it all with 2-space indent.

Reply:
Interesting, I didn't realize they were tabs. I don't use tabs. Will check later.